client/packages/config/src/config.ts

```diff
@@ -25,15 +20,14 @@ const productionApiHost = `${protocol}//${hostname}:${port}`;
 const developmentApiHost =
   (typeof API_HOST !== 'undefined' && API_HOST) || defaultDevelopmentApiHost;
 const apiHost = isProductionBuild ? productionApiHost : developmentApiHost;
-const isPluginLocal = typeof PLUGIN_URL !== 'undefined' && !!PLUGIN_URL;
-const pluginUrl = isPluginLocal ? PLUGIN_URL : `${apiHost}/plugins`;
+
+const pluginUrl = `${apiHost}/plugins`;
```

Comment:
am using the same format (`[file]`.js) when loaded locally or remotely, so have removed the extension part